validator/src/cli.rs

```diff
@@ -1304,6 +1305,15 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .long("replay-slots-concurrently")
                 .help("Allow concurrent replay of slots on different forks")
         )
+        .arg(
+            Arg::with_name("banking_trace_size")
+                .long("enable-banking-trace")
```

Comment:
Think this got missed when changing from enabled to size

Reply:
actually, it's not stale code. hopefully, this commit clarifies: db4732c1efec119e47e23e880bf2c7b8e1742d04